books/books/settings.py

```diff
@@ -0,0 +1,47 @@
+BOT_NAME = 'books'
+
+SPIDER_MODULES = ['books.spiders']
+NEWSPIDER_MODULE = 'books.spiders'
+
+# some sane limits by default (override if needed)
+CLOSESPIDER_ITEMCOUNT = 1000
+#CLOSESPIDER_TIMEOUT = 20
+#DOWNLOAD_DELAY = 0.25
+RETRY_ENABLED = False
+COOKIES_ENABLED = False
```

Comment:
Maybe let's enable cookies? Unless I misunderstood, there is some code in the spider that deals with them.